semantics/executable-spec/src/Control/State/Transition/Extended.hs

```diff
@@ -431,14 +462,12 @@ applySTS = applySTSOptsEither defaultOpts
     defaultOpts =
       ApplySTSOpts
         { asoAssertions = globalAssertionPolicy,
-          asoValidation = ValidateAll
+          asoValidation = ValidateAll,
+          asoEvents = EPDiscard
         }
 
-#ifdef STS_ASSERT
```

Comment:
If `STS_ASSERT` flag is no longer used we might want to also remove it from the cabal file

Reply:
Oop, good catch, not sure how this got deleted. Totally unintentional!